### DISCUSSION TITLE
can i set  server if the ip is unknown?

### DISCUSSION THREAD

**Discussion by TinyBrickBoy** [haveachin/infrared#217] [Q&A]
can i set  server if the ip is unknown?

**Comment by GomorrhaDev** [comment]
What do you mean with "if the ip is unknown"

**Comment by TinyBrickBoy** [comment]
if you like connect using a ip with is not set in a proxy config. i want to make a backup server

**Comment by TinyBrickBoy** [comment]
?

**Comment by nnitch** [comment]
Correct me if I'm wrong, but you want to have a default server that players connect to if the hostname they are using to connect isn't used in another proxy configuration? I don't think this is possible with infrared just yet. I haven't used Infrared in a while so take this advice with a grain of salt.
